iOS/CodiumUnWrapper/README.md

```diff
@@ -0,0 +1,12 @@
+# Description
+
+CodiumUnWrapper will help you to safe unwrap Optionals 
+
+# Supported value Types
+
+CodiumUnWrapper supports most of the famous value types such as String, Int, Bool, Double, Float, Array, Dictionary and ... .
+It can also check if the optional value is nil or not 
+
+# Extension
+
+to use CodiumUnWrapper you just need to copy the extension file in your app and use `unWrap` or `unWrapOr(yourDefaultValue)`  for unWrap Optionals . or if you just want to check the Optional value is nil or not you can use  `isNil` or `isNotNil` to check that. 
```

Comment:
To use this extension you only need to copy the example extension code to your project.

Reply:
Then you can give example of how to use it. Example:
```swift
let value: String?
let result = value.unwrap
```